src/main/java/seedu/address/storage/JsonAdaptedTask.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.task.DateTimeDue;
+import seedu.address.model.task.Description;
+import seedu.address.model.task.Task;
+
+/**
+ * Jackson-friendly version of {@link Task}.
+ */
+public class JsonAdaptedTask {
```

Comment:
let us know if you need help with writing json test cases! Although Json is unfamiliar to me too, i'm sure we can work something out